### PR TITLE
PEP8 \o/

### PR DESCRIPTION
**Pull request by Rafiot**
Step 1: make sure my pep8 config stops whining.

WIP, do not merge immediately :)

### CODE REVIEW

File: threatintel/converters/buildMISPAttribute.py [hunk 1]
```diff
@@ -108,16 +106,16 @@ def buildAttribute(pkg, mispEvent, mispAPI):
                             mispAPI.add_email_dst(mispEvent, obj.header.to, comment="Email to Addr")
                         if obj.header.subject:
                             mispAPI.add_email_subject(mispEvent, obj.header.subject, comment="Email Sub")
-                    if emsg.attachments:
+                    if emsg.attachments:  # FIXME Broken
```
Comment (Rafiot) [inline threatintel/converters/buildMISPAttribute.py]: @FloatingGhost `emsg` isn't defined, should it be `obj`?

Comment (FloatingGhost) [inline threatintel/converters/buildMISPAttribute.py]: Yup, seems like it should be

Comment (Rafiot) [inline threatintel/converters/buildMISPAttribute.py]: Fixed it :)

**Comment by Rafiot** [comment]
@FloatingGhost if you can have a short look at the changes, I think it will be fine (and the tests are passing \o/).

**Pull request merged**